### PR TITLE
Removed starter HTML and added header bar + interactive button links

### PR DESCRIPTION
**Pull request by Mmendero**
Changes:
- Removed SPS starter code for webapp
- Added HTML code to have a header bar for the website
- Added styles to CSS file for the new HTML elements
- Added Project ID to pom.xml file
- Renamed "portfolio" folder

### CODE REVIEW

File: Site_Files/pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>mmendero1-sps-summer20</deploy.projectId>
```
Comment (perrycate) [inline Site_Files/pom.xml]: Probably ought to have this mention team3 and/or everyone's usernames.

File: Site_Files/src/main/webapp/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  </head>
+
+  <body>
+
+    <div class="header">
+        <a href="#default" class="logo">Resource Site</a>
+        <div class="header-right">
+            <a class="active" href="#home">Home</a>
+            <a href="#contact">Contact</a>
+            <a href="#login-signup">Login/Signup</a>
```
Comment (matthewtlam) [inline Site_Files/src/main/webapp/index.html:17]: We can change this to be two different links where a different one can appear depending on who is signed in later

**Review by matthewtlam — APPROVED** [review]

File: Site_Files/src/main/webapp/index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
```
Comment (ben188ben188) [inline Site_Files/src/main/webapp/index.html:7]: test code review

Comment (ben188ben188) [reply Site_Files/src/main/webapp/index.html:7]: add somthing

**Comment by ben188ben188** [comment]
approved

**Review by ben188ben188 — APPROVED** [review]

**Pull request merged**